includes/Modules/Analytics_4.php

```diff
@@ -493,11 +534,27 @@ function( $property ) {
 				return array_map( array( self::class, 'filter_webdatastream_with_ids' ), $webdatastreams );
 			case 'GET:webdatastreams-batch':
 				return self::parse_webdatastreams_batch( $response );
+			case 'GET:container-lookup':
+				return (array) $response;
```

Comment:
Here we shouldn't be casting the response to an array – doing so exposes some private properties on the response that we shouldn't. In this case we can simply defer to the default behavior.

```suggestion
```

Reply:
Updated.